teamsupport/models.py

```diff
@@ -23,6 +27,27 @@ def __init__(self, client, ticket_id=None, data=None):
                 '(neither given)')
         self.id = self.TicketID
 
+    @classmethod
+    def create(cls, client, first_name, email, category, name, description):
```

Comment:
not related to your changes, but passing client everywhere looks unnecessary.  I would create  `get_teamsupport_service()` classmethod in  `XmlMolel` for that.